### PR TITLE
Adds files and folders created by setup.py

### PR DESCRIPTION
**Pull request by bjcoleman**
On my system I was getting additional files and folders:
* *.egg for each of package
* build/ for each package

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ __pycache__/
 .eggs/
 build/
 *egg-info/
+*.egg
+dist/
```
Comment (sommern) [inline .gitignore]: No newline at the end. I don't think it matters to git but it's good practice and avoids git notifying you about it when doing diffs.

**Comment by bjcoleman** [comment]
fixed.  Not sure why I didn't hit return at the end.  Not something I usually do.

**Comment by sommern** [comment]
Needs to merge into develop, not master.

**Comment by sommern** [comment]
Use imperative for commit messages: Add, not Adds

**Comment by bjcoleman** [comment]
Closing because you cannot change the target branch.  See isaacs/github/#18.

**Pull request closed without merging**